src/components/AttachmentPicker/index.native.js

```diff
@@ -51,6 +52,24 @@ function showDocumentPicker(callback) {
 function show(callback) {
     RNImagePicker.showImagePicker(imagePickerOptions, (response) => {
         if (response.error) {
+            if (response.error === 'Camera permissions not granted') {
```

Comment:
Some additional context on the unconventional error check here:

Since we're using the deprecated 2.x.x `react-native-image-picker` module at the moment, we're forced to use the [textual `error` message](https://github.com/react-native-image-picker/react-native-image-picker/blob/2.3.4/docs/Reference.md#the-response-object) returned [by the library](https://github.com/c19354837/react-native-image-picker/blob/575d5b12693e81d7c1ea618998798761d14e670b/ios/ImagePickerManager.m#L204).

The new  3.x.x version has a [dedicated error code for permissions](https://github.com/react-native-image-picker/react-native-image-picker#ErrorCode), which is a far cleaner solution.

@AndrewGable Are there any plans for us to upgrade `react-native-image-picker`, or should this solution be acceptable for now? Thanks!

Reply:
I think updating to the latest would be great 👍